.teamcity/src/main/kotlin/common/performance-test-extensions.kt

```diff
@@ -26,6 +26,8 @@ fun BuildType.applyPerformanceTestSettings(os: Os = Os.LINUX, timeout: Int = 30)
     applyDefaultSettings(os = os, timeout = timeout)
     artifactRules = """
         build/report-*-performance-tests.zip => .
+        build/report-*-performance.zip => $hiddenArtifactDestination
```

Comment:
Why do you move that now to the hidden artifact location as well? This shouldn't contain any secrets and contributors should be able to look at the performance report.

Reply:
The contributors can already see the performance report in `build/report-*-performance-tests.zip`, I'm just adding something that was not published before (e.g. TD `trace.json`). I don't have any preference, just keeping it same as before: https://github.com/gradle/gradle/blob/ebcba5aa96039ef3a65cb1c1fbe7eb2b260dcc00/.teamcity/src/main/kotlin/common/extensions.kt#L81

Reply:
Ok, got it!